Add local up install doc (#91)
kubeedge/sedna: docs/setup/local-up.md

### Deploy local Sedna clusters

> Once you have docker running you can create a local Sedna cluster with:

Reviewer: ```suggestion
Once you have docker running, you can create a local Sedna cluster with:

```

```
bash scripts/local-up.sh
```
This script uses [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) to create a 

> local k8s cluster, and boot a KubeEdge cluster with one master node and one edge node.

Reviewer: ```suggestion
local k8s cluster with one master node, and boots one edge node by running KubeEdge.
```

### Deploy local Sedna clusters

> Once you have docker running you can create local Sedna clusters with:

Reviewer: Add a description such as it uses kind for creating a k8s cluster, and a link for kind

Author: ok

```
bash scripts/local-up.sh
```

> You can watch clusters with:

Reviewer: ```suggestion
This script will boot a k8s cluster with one master node and a edge node, you can see them using `kubectl`:

```

Author: k8s -> KubeEdge
